mappings/net/minecraft/structure/MineshaftGenerator.mapping

```diff
@@ -93,6 +104,18 @@ CLASS net/minecraft/class_3353 net/minecraft/structure/MineshaftGenerator
 			ARG 4 maxX
 			ARG 5 y
 			ARG 6 z
+		METHOD method_33778 containsLiquid (Lnet/minecraft/class_1922;Lnet/minecraft/class_3341;)Z
+			ARG 1 world
+			ARG 2 box
+		METHOD method_33779 isAir (Lnet/minecraft/class_1922;Lnet/minecraft/class_2338$class_2339;III)Z
+			COMMENT Determines whether the block at the given coordinates is air.
+			COMMENT
+			COMMENT <p>This method mutates the passed position by {@linkplain net.minecraft.util.math.BlockPos.Mutable#set(int, int, int) setting it} based on the integer coordinates.
```

Comment:
```suggestion
			COMMENT @apiNote The {@code pos} will be {@linkplain net.minecraft.util.math.BlockPos.Mutable#set(int, int, int) set} to the given x, y, z to avoid object allocation.
```